sdk/src/Services/SQS/Custom/Model/GetQueueAttributesResponse.extensions.cs

```diff
@@ -178,6 +172,44 @@ public string Policy
             }
         }
 
+        /// <summary>
+        /// Whether or not the queue is a First-In-First-Out (FIFO) queue
+        /// </summary>
+        public bool FifoQueue
+        {
+            get
+            {
+                var attributeValue = getAttributeValue(SQSConstants.ATTRIBUTE_FIFO_QUEUE);
+
+                if (attributeValue == null || attributeValue.Trim() == string.Empty)
+                {
+                    return false;
+                }
+
+                bool result;
+                return bool.TryParse(attributeValue, out result);
```

Comment:
Ah! My bad. I completely missed that. Thanks for pointing that out. Rectified in	816b607.